actor-typed-tests/src/test/scala/docs/org/apache/pekko/typed/MailboxDocSpec.scala

```diff
@@ -21,6 +21,7 @@ import pekko.actor.typed.Behavior
 import pekko.actor.typed.MailboxSelector
 import pekko.actor.typed.scaladsl.Behaviors
 import com.typesafe.config.ConfigFactory
+import org.apache.pekko.actor.typed.Dispatchers
```

Comment:
can you remove the org.apache from this import?

Reply:
Fixed, Thanks.